code/game/objects/structures/window.dm

```diff
@@ -442,7 +441,7 @@
 	heat_resistance = 50000
 	armor = list("melee" = 85, "bullet" = 20, "laser" = 0, "energy" = 0, "bomb" = 60, "bio" = 100, "rad" = 100, "fire" = 99, "acid" = 100)
```

Comment:
This has less melee armor than the other two. Oversight or is there some reason behind it?